### DISCUSSION TITLE
UI mysteriously changed to German!

### DISCUSSION THREAD

**Discussion by cyprian49** [jamulussoftware/jamulus#1278] [Technical Support & Help] [answered]
I used Jamulus this afternoon and the UI was English, as it's been since forever. Tonight, I opened the program and everything is in German. What to do?

**Comment by ann0see** [comment, marked as answer]
That's strange. Which version are you running? 

In the meantime, you can just change to English in settings.

Edit: This can be done if you click on "Einstellungen" and change the language on the right side of the settings Window. (Version 3.6.2)

**Reply by cyprian49** [reply]
Whatever is latest. I updated thinking that might resolve the problem. Sadly, I do not know where to find the language setting. Can you enlighten me?

**Reply by ann0see** [reply]
You can change it if you click on "Einstellungen"

**Reply by cyprian49** [reply]
Yes, that worked. Changed Language setting, closed + reopened app. Thanks so much!

**Reply by ann0see** [reply]
Great! It would be good to know what the problem was initially
